src/Action/PlaceholderAction.php

```diff
@@ -21,11 +21,11 @@
 final class PlaceholderAction
 {
     /**
-     * @param object $data
+     * @param mixed|null $data
```

Comment:
```suggestion
     * @param mixed $data
```

`mixed` arelady includes `null`.